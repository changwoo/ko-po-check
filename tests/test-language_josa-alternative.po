# -*-text-*-
# test 2 - 조사 구별
#
# * printf()류 함수의 format string에서 format 다음에 오는 조사는
# 받침이 있는 경우와 없는 경우 모두를 생각해야 한다.
#
msgid ""
msgstr ""
"Project-Id-Version: test 2\n"
"POT-Creation-Date: 2002-07-24 22:09+0900\n"
"PO-Revision-Date: 2002-07-24 22:09+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

# E:조사 구별하지 않은 경우
#: test.c:2
#, c-format
msgid "%s is ..."
msgstr "%s는 ..."

# E:조사 구별하지 않은 경우
# - 따옴표와 같이 쓴 경우
#: test.c:2
#, c-format
msgid "'%s' is ..."
msgstr "'%s'는 ..."

# E:조사 구별하지 않은 경우
# - 띄어 쓴 경우
#: test.c:2
#, c-format
msgid "Making %s..."
msgstr "%s 을 만들고 있습니다..."

# E:조사 구별하지 않은 경우
# - 따옴표 + 띄어 쓴 경우
#: test.c:2
#, c-format
msgid "Making '%s'..."
msgstr "'%s' 를 만들고 있습니다..."

# 올바른 조사 구별
#: test.c:2
#, c-format
msgid "What is %s"
msgstr "%s이(가) 무엇일까"

# E: 각종 format type
#: test.c:2
#, c-format
msgid ""
"%d is, %i is, %o is, %u is, %f is, %e is, %E is, %g is, %G is, %a is,\n"
"%A is, %c is, %C is, %s is, %p is, %n is, %m is, %h is, %j is, %l is,\n"
"%L is, %q is, %t is, %x is, %X is, %z is, %Z is"
msgstr ""
"%d는, %i는, %o는, %u는, %f는, %e는, %E는, %g는, %G는, %a는,\n"
"%A는, %c는, %C는, %s는, %p는, %n은, %m은, %h는, %j는, %l은,\n"
"%L은, %q는, %t는, %x는, %X는, %z는, %Z는"

# E: parameter number
#: test.c:2
#, c-format
msgid "%s is %s"
msgstr "%2$s는 %1$s입니다"

# E: integer modifier
#: test.c:2
#, c-format
msgid "%04d is, %+8d is, %-8d is, %-08d is, %+08d is, %8.3d is, %-8.3d is, %#x is"
msgstr "%04d는, %+8d는, %-8d는, %-08d는, %+08d는, %8.3d는, %-8.3d는, %#x는"

# E: l,h, modifier
#: test.c:2
#, c-format
msgid "%4ld is, %8ls is, %lc is, %hf is"
msgstr "%4ld는, %8ls는, %lc는, %hf는"

# 조사가 아니다
#: test.c:2
#, c-format
msgid "%s silver medal"
msgstr "%s 은메달"

# 조사가 아니다
#: test.c:2
#, c-format
msgid "it is not %s"
msgstr ""
"%s은\n"
"아닙니다"

# Python
#: test.py:2
#, python-format
msgid "%(name)s is ..."
msgstr "%(name)s는 ..."

# Python
#: test.py:2
#, python-format
msgid "'%(num)03d' is ..."
msgstr "'%(num)03d'는 ..."

# Python
#: test.py:2
#, python-format
msgid "'%(num)03d' is what ..."
msgstr "'%(num)03d'은(는) 무엇일까 ..."
