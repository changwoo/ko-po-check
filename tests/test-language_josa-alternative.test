#!/bin/sh

FILTERS="language/josa-alternative"

DIR="`dirname $0`"
NAME="`basename $0 .test`"
BUILDDIR=`pwd`
POFILE=$NAME.po
OUTFILE=$NAME.out

TFM=`mktemp`
export PYTHONPATH=..
cat $POFILE | python3 $BUILDDIR/../scripts/ko-po-check --filters="${FILTERS}" - > $TFM
diff -u $TFM $OUTFILE
RESULT=$?
rm -f $TFM
exit $RESULT
