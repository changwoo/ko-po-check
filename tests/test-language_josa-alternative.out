-:22: "%s는": 받침에 따른 조사 구별이 없습니다. '%s은(는)'
-:29: "%s'는": 받침에 따른 조사 구별이 없습니다. '%s'은(는)'
-:36: "%s 을": 받침에 따른 조사 구별이 없습니다. '%s을(를)'
-:43: "%s' 를": 받침에 따른 조사 구별이 없습니다. '%s'을(를)'
-:58: "%Z는": 받침에 따른 조사 구별이 없습니다. '%Z은(는)'
-:67: "%2$s는": 받침에 따른 조사 구별이 없습니다. '%2$s은(는)'
-:73: "%#x는": 받침에 따른 조사 구별이 없습니다. '%#x은(는)'
-:79: "%hf는": 받침에 따른 조사 구별이 없습니다. '%hf은(는)'
-:91: "%s은": 받침에 따른 조사 구별이 없습니다. '%s은(는)'
-:99: "%(name)s는": 받침에 따른 조사 구별이 없습니다. '%(name)s은(는)'
-:105: "%(num)03d'는": 받침에 따른 조사 구별이 없습니다. '%(num)03d'은(는)'
